### PR TITLE
Improve Alfafile download functionality.

### PR DESCRIPTION
**Pull request by axzxc1236**
I tinkered with the code to make it usable.
I am not sure I coded it right as I didn't take my time to fully understand Pyload's code architecture, but it's "usable" for me.

### CODE REVIEW

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5668598&bid=18556973)

File: module/plugins/hoster/AlfafileNet.py [hunk 1]
```diff
@@ -31,7 +31,12 @@ class AlfafileNet(SimpleHoster):
 
     LINK_PATTERN = r'<a href="(.+?)" class="big_button"><span>Download</span></a>'
 
-    DL_LIMIT_PATTERN = r'Try again in (.+?)<'
+    DL_LIMIT_PATTERN = "You can't download not more than"
+
```
Comment (vuolter) [inline module/plugins/hoster/AlfafileNet.py:35]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5668598)

**Comment by GammaC0de** [comment]
@axzxc1236 
First of all, `multiDL` is disabled by default for free users
so  `multiDL = False` is unnecessary here.

Secondly, you said:
> the server doesn't tell us how much time do we need to wait

But I do see how much time to wait:
Capture

Am I missing something?

**Comment by axzxc1236** [comment]
hmm.... weird, when I tried to debug my issues it just told me I can't download more than one file at a time. (that what caused my pyload instance doesn't do retries)

I will do more tests to check my findings.

**Comment by axzxc1236** [comment]
> When this happens, can you open the URL in your browser and see what message do you get?  
Localized message saying the exact same thing, no count down.  
[html.zip](https://github.com/pyload/pyload/files/4765795/html.zip)

I did a **fresh install** (zero code changes, stable branch) of Pyload on a VPS **with dedicated IP**, exact same thing happened.

https://i.imgur.com/uevZznf.png

**Comment by GammaC0de** [comment]
After how many successful downloads the plugin start to fail?

**Comment by axzxc1236** [comment]
One, exactly one, everytime.

**Should I** close this pull request and open an issue instead?

Because for now I just setup a desktop environment on my Linux system and use JDownloader instead, JDownloader also only downloaded exactly one file overnight.

**Comment by GammaC0de** [comment]
Yes, Please open an issue

**Comment by GammaC0de** [comment]
Plugin updated again, please test.

**Pull request closed without merging**